pages/account.js

```diff
@@ -37,116 +52,112 @@ export default function Account() {
+    <section className="bg-black mb-32">
+      <div className="max-w-6xl mx-auto pt-8 sm:pt-24 pb-8 px-4 sm:px-6 lg:px-8">
+        <div className="sm:flex sm:flex-col sm:align-center">
+          <h1 className="text-4xl font-extrabold text-white sm:text-center sm:text-6xl">
+            Account
+          </h1>
+          <p className="mt-5 text-xl text-accents-6 sm:text-center sm:text-2xl max-w-2xl m-auto">
+            We partnered with Stripe for a simplified billing.
+          </p>
+        </div>
+      </div>
+      <div className="p-4">
+        <Card
+          title="Your Plan"
+          description={
+            subscriptionName &&
+            `You are currently on the ${subscriptionName} plan.`
+          }
+          footer={
+            <div className="flex items-start justify-between flex-col sm:flex-row sm:items-center">
+              <p className="pb-4 sm:pb-0">
+                Manage your subscription on Stripe.
               </p>
               <Button
                 variant="slim"
-                disabled={loading || !subscriptions?.length}
+                loading={checkoutLoading}
+                disabled={checkoutLoading || !subscriptions?.length}
                 onClick={redirectToCustomerPortal}
               >
-                Access the customer portal
+                Open customer portal
               </Button>
             </div>
+          }
+        >
+          <div className="text-xl mt-8 mb-4 font-semibold">
+            {loading ? (
+              <div className="h-12 mb-6">
+                <LoadingDots />
+              </div>
+            ) : subscriptionPrice ? (
+              `${subscriptionPrice}/month`
+            ) : (
+              <Link href="/">
+                <a>See pricing</a>
+              </Link>
+            )}
           </div>
-        </div>
+        </Card>
+        <Card
+          title="Your Name"
+          description="Please enter your full name, or a display name you are comfortable with."
+          footer={<p>Please use 64 characters at maximum.</p>}
+        >
+          <div className="text-xl mt-8 mb-4 font-semibold">
+            {userDetails ? (
+              <Input
+                type="text"
+                placeholder="Guillermo Rauch"
+                defaultValue={
+                  userDetails
+                    ? `${userDetails.first_name} ${userDetails.last_name}`
+                    : undefined
+                }
+                className="rounded-md text-base max-w-sm p-0"
+              />
+            ) : (
+              <div className="h-8 mb-6">
+                <LoadingDots />
+              </div>
+            )}
+          </div>
+        </Card>
+        <Card
+          title="Your Email"
+          description="Please enter the email address you want to use to login."
+          footer={<p>We will email you to verify the change.</p>}
+        >
+          <p className="text-xl mt-8 mb-4 font-semibold">
+            <Input
```

Comment:
@leerob are we planning to implement the database actions for update the user data? I'd default to just displaying them instead. Wdyt? 

cc @kiwicopple

Reply:
We could just disable the inputs. I like having them because people will likely need them.